main.go

```diff
@@ -32,8 +35,14 @@ func init() {
 		fmt.Println(app.Name)
 		fmt.Println(c.String("command"))
 
+		ex, err := os.Executable()
+		if err != nil {
+			panic(err)
+		}
+		exPath := filepath.Dir(ex)
+
 		var savedComs savedCommands
-		data, err := ioutil.ReadFile("saved_commands.json")
+		data, err := ioutil.ReadFile(filepath.Join(exPath,"saved_commands.json"))
```

Comment:
This is minor, but there should be a space between the arguments here.

However, that leads me to wonder: did you not run `gofmt` on this code?  Could you ensure `gofmt` is always run on code you submit?  You should be able to just integrate it into your IDE.

Reply:
I didn't catch this before, but this actually changes my original intention for where to look for that file. It is supposed to look in the current directory for the file, and not side-by-side with the executable itself.

Oh, and looking up the function reminded me that `ioutil.ReadFile()` is deprecated and should be replaced with `os.ReadFile()`, so we should probably do that.

Reply:
(deleted comment because I was confused)

Reply:
It seems the VS code plugin for Go stopped supporting this version of go (1.17) and stopped running gofmt. Should I update the go version?

Reply:
Yeah, while you're here updating to something less ancient is probably a good idea. :slightly_smiling_face:

Reply:
Updated to 1.20